src/users.ts

```diff
@@ -48,6 +48,8 @@ interface CreateUserParams {
   password: string,
   first_name: string,
   last_name: string,
+  birthday?: string,
+  phone_numbers?: Array<{number: string}>
```

Comment:
We don't need it right now, but it's interesting that in malan, you can pass in any attribute on create, but here you can only pass in a few. We could consider adding them?

Reply:
Yeah!  No reason we can't add them.  The only reason they aren't in libmalan is because nobody was using libmalan at the time those were added to malan.  We should add everything!

Reply:
On that note there are a few endpoints in malan that aren't in libmalan either.  I'll work on getting a list of all the things missing.

Reply:
I'll add them in another PR later this week!